app/jobs/cloud_fixity_job.rb

```diff
@@ -9,6 +9,7 @@ def perform(status:, resource_id:, child_property:, child_id:)
     @resource_id = resource_id
     @child_property = child_property
     @child_id = child_id
+    return unless resource_exist?
```

Comment:
Question: What's the reasoning behind an early return versus a rescue for the perform method?

Reply:
The perform method is already pretty long. You'd have to shove the same into it as in the `resource_exist?` method.